x/il/keeper/abci.go

```diff
@@ -152,10 +173,6 @@ func (k Keeper) EndBlocker(ctx sdk.Context) {
 			)
 		}()
 
-		// TODO: technically we should remove the stoploss position now that is has been executed, but we still need to account
-		// for the cases when the outgoing txs to ethereum fail for some reason.
-		k.DeleteStoplossPosition(ctx, address, stoploss.UniswapPairID)
```

Comment:
seems like unless the user deletes the stop loss we wont prune it?

Reply:
correct, this has some tradeoffs. For example, if the IL is decreasing all the time the position will be sitting around for a long time. Ideally, once we implement the hooks on gravity, we can automatically delete the position once we know it was executed on ethereum.